### PR TITLE
docs(website): created a discover Podman Desktop page

### PR DESCRIPTION
**Pull request by shipsing**
Signed-off-by: Shipra Singh <shipsing@redhat.com>

### What does this PR do?
Add a discover Podman Desktop page to the website
### Screenshot / video of UI

https://670f55afe49bd8ea027dc6f1--podman-desktop-pr.netlify.app/docs/discover-podman-desktop
### What issues does this PR fix or reference?
https://github.com/containers/podman-desktop/issues/5369

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: website/docs/discover-podman-desktop.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+The key differentiator is its intuitive UI interface that allows you to visualize your development environment. Be it the number of pods or Kubernetes clusters running or the number of deployments available or the number of services exposed, you can visualize your dev environment with a click. In addition, you have the option of running this tool on three different operating systems; macOS, Windows, and Linux. Podman Desktop runs your workloads on a Podman engine and therefore, provides you a Podman-native capabilities to interact with your applications. But, this does not mean you cannot run your Docker applications. You can use the Docker compatibility UI feature for a smooth transition to Podman Desktop.
+
+Let’s quickly explore this tool’s key functionalities.
```
Comment (axel7083) [inline website/docs/discover-podman-desktop.md]: I must be honest, if I open a documentation with that much text I will not be reading it. The content is interesting, but maybe too compact as an introduction

File: website/docs/discover-podman-desktop.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+sidebar_position: 2
+title: Discover Podman Desktop
+description: Discovering the Podman Desktop application
+keywords: [podman desktop, podman, Discover features]
+tags: [podman-desktop, discover-podman-desktop, features]
+---
+
+# Discover Podman Desktop
+
+Podman Desktop is a community-driven tool that is an alternative to Podman and Docker tools. You can perform a wide range of basic and advanced tasks. From creating a container or pod to setting up a kubernetes cluster to running an AI application to integrating your local tools using extensions, Podman Desktop offers all these features to ease your daily development tasks. But, what makes Podman Desktop different from other similar community-driven tools?
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]: Podman Desktop is not an alternative to Podman it is a developer tool on top of Podman

File: website/docs/discover-podman-desktop.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+The key differentiator is its intuitive UI interface that allows you to visualize your development environment. Be it the number of pods or Kubernetes clusters running or the number of deployments available or the number of services exposed, you can visualize your dev environment with a click. In addition, you have the option of running this tool on three different operating systems; macOS, Windows, and Linux. Podman Desktop runs your workloads on a Podman engine and therefore, provides you a Podman-native capabilities to interact with your applications. But, this does not mean you cannot run your Docker applications. You can use the Docker compatibility UI feature for a smooth transition to Podman Desktop.
+
+Let’s quickly explore this tool’s key functionalities.
+
+#### Manage containerized environment
+
+![containers](img/containers-component.png)
+
+Access the **Containers** component in the UI to do the container-related tasks:
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]: Maybe page would be better than component ?

File: website/docs/discover-podman-desktop.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+- Create a container with a container or docker file or from an existing image.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
- Create a container with a Containerfile or Dockerfile or from an existing image.
```

File: website/docs/discover-podman-desktop.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+- Create a pod with selected containers.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
- Create a pod from a list of containers.
```

File: website/docs/discover-podman-desktop.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+- [Deploy a container to a Kubernetes cluster](/docs/kubernetes/deploying-a-container-to-kubernetes).
+- Search, start, restart, stop, or delete a container.
+- Bulk deletion of selected containers.
+- Export a container to your local machine.
+- View the summary and logs of a container.
+- View the kube configuration of the container if it is a Kubernetes resource.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]: View the Kubernetes pod definition of the container

File: website/docs/discover-podman-desktop.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+- Interact with the container using a terminal.
+- View containers based on their status, whether they are running or stopped.
+- Remove all unused containers from the Podman engine for efficient memory usage.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]: That is not restricted to Podman

File: website/docs/discover-podman-desktop.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+For more details, refer to [Working with containers](/docs/containers).
+
+#### Podify your containers
+
+![pods](img/pods-component.png)
+
+Access the **Pods** component in the UI to do the pod-related tasks:
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]: Same remark

File: website/docs/discover-podman-desktop.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+
+- Create a pod from a Kubernetes YAML file.
+- Search, start, restart, stop, or delete a pod.
+- View the status of containers that are part of the pod, whether they are running, created, exited, or in waiting state.
+- View the summary and logs of the pods
+- Generate and view configuration for a Kubernetes pod.
+- [Deploy a pod to a Kubernetes cluster](/docs/kubernetes/deploying-a-pod-to-kubernetes).
+- Interact with the pod using a terminal.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
- Interact with the pod containers using a terminal.
```

File: website/docs/discover-podman-desktop.md [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+- View pods based on their status, whether they are running or stopped.
+- Remove all unused pods from the Podman engine for efficient memory usage.
+
+#### Manage images
+
+![images](img/images-component.png)
+
+Access the **Images** component in the UI to do the image-related tasks:
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]: Same remark

File: website/docs/discover-podman-desktop.md [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+
+- Build an image from a container or docker file.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
- Build an image from a Containerfile or Dockerfile.
```

File: website/docs/discover-podman-desktop.md [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+- Pull an image from a registry.
+- Search or delete an image.
+- Create a container from the image.
+- Push an image to a configured registry.
+- Edit the image name and tag.
+- View the image history.
+- Save an image to your local machine.
+- Push an image to a cluster.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
- Push an image to a cluster internal registry.
```

File: website/docs/discover-podman-desktop.md [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+- Import images from your local machine into the Podman Desktop application.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
- Import images from your local machine into the container engine.
```

File: website/docs/discover-podman-desktop.md [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+
+#### Manage Kubernetes-based environment
+
+![kubernetes](img/kubernetes-component.png)
+
+Access the **Kubernetes** component in the UI to do the Kubernetes-related tasks:
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]: Same remark

File: website/docs/discover-podman-desktop.md [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+
+- Create an object after applying a kubernetes YAML configuration.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
- Create a Kubernetes resource after applying a Kubernetes YAML configuration.
```

File: website/docs/discover-podman-desktop.md [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+- Search or delete an object.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
- Search or delete a Kubernetes resource.
```

File: website/docs/discover-podman-desktop.md [hunk 17]
```diff
@@ -0,0 +1,137 @@
...
+- View the summary and Kubernetes configuration of the object
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
- View the summary and Kubernetes configuration of the Kubernetes resource
```

File: website/docs/discover-podman-desktop.md [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+- Edit the configuration of the Kube object and apply those changes to your cluster directly from the UI.
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
- Edit the configuration of the Kubernetes resource and apply those changes to your cluster directly from the UI.
```

File: website/docs/discover-podman-desktop.md [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+
+For more details, refer to [From containers to Kubernetes](/docs/kubernetes).
+
+#### Manage Docker compatibility
+
+Docker compatibility is a way to configure an environment in which you can run your Docker applications, commands, and tools on a Podman engine without reconfiguration. It encompasses two stages. You first [import your saved containers](/docs/migrating-from-docker/importing-saved-containers) into Podman Desktop using CLI and then access the [Docker Compatibility settings](/docs/migrating-from-docker/managing-docker-compatibility) to:
+
+- Check the system socket mapping status
+- On macOS: Use all Docker tools, including CLI, with the Podman engine
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]: Why is it restricted to MacOS ?

File: website/docs/discover-podman-desktop.md [hunk 20]
```diff
@@ -0,0 +1,137 @@
...
+- Use the Podman CLI to run Compose applications
+- Select a Docker CLI context
+
+On macOS, consider installing the [podman-mac-helper tool](/docs/migrating-from-docker/using-podman-mac-helper) to use Docker commands with a Podman engine.
+
+#### Integrate your tools using extensions
+
+![extensions](img/extentions-component.png)
+
+Podman Desktop provides a wide range of extensions that can be used to integrate your local tools with Podman Desktop. After installing the required extension, you can do development tasks like creating a Kube cluster, creating an AI application, or creating a bootable container. List of extensions available:
```
Comment (jeffmaury) [inline website/docs/discover-podman-desktop.md]:
```suggestion
Podman Desktop provides a wide range of extensions that can be used to integrate your local tools with Podman Desktop. After installing the required extension, you can do development tasks like creating a Kubernetes cluster, creating an AI application, or creating a bootable container. List of extensions available:
```

File: website/docs/discover-podman-desktop.md (hunk 19 above)
Comment (shipsing) [reply website/docs/discover-podman-desktop.md]: Right now, this capability is not available on Windows. See this UX mockup screenshot: https://github.com/containers/podman-desktop/issues/8338#issuecomment-2367822941

The Third-Party Docker Tool Compatibility setting is only available on macOS, not on Windows.

File: website/docs/discover-podman-desktop.md (hunk 1 above)
Comment (shipsing) [reply website/docs/discover-podman-desktop.md]: @axel7083, Thank you for being honest :)
If you look at the content structure in entirety, the subsequent sections discuss about key features in brief. We have not built much context there. So, having a bit of context in the beginning would help readers to relate with the key functionalities of Podman Desktop described later.  
Also, you would come to this exploring page mostly if you are a new user to Podman Desktop. So, buiding a bit of context in the beginning would help them in understanding the purpose of the PD tool.

**Comment by shipsing** [comment]
Thank you @axel7083 @jeffmaury for your suggestions. Could you please verify the PR and confirm? Thanks

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**